runtime/container_linux.go

```diff
@@ -284,6 +284,27 @@ func (c *container) Stats() (*Stat, error) {
 	}, nil
 }
 
+// Status implements the runtime Container interface.
+func (c *container) Status() (State, error) {
+	args := c.runtimeArgs
+	args = append(args, "state", c.id)
+
+	out, err := exec.Command(c.runtime, args...).CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf(string(out))
+	}
+
+	// We may not know the exactly structure of the response if we
```

Comment:
Hum, although not defined by the spec. `containerd` would requires all runtimes to have at least the Status field.

Maybe change the comment to: `We only require the runtime json output to have a top level Status field`